scaffolding-chef/lib/scaffolding.ps1

```diff
@@ -115,13 +141,10 @@ log_level = "warn"
 env_path_prefix = ";C:/WINDOWS;C:/WINDOWS/system32/;C:/WINDOWS/system32/WindowsPowerShell/v1.0;C:/ProgramData/chocolatey/bin"
 ssl_verify_mode = ":verify_peer"
 
-[chef_license]
-acceptance = "undefined"
-
-[data_collector]
-enable = false
```

Comment:
I think this needs to get updated in the `do_default_build()` as well so that if there is an automate url or token then the datacolletor is added to the `client-config.rb` properly.

Reply:
Thank you! Fixed <3